matchya-web/src/data/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Score } from '../types';
+
+export const criteria: string[] = [
+    'Knows Python',
+    'Used Django before'
+];
+
+export const candidates: Score[] = [
```

Comment:
Let's name the variables with the prefix of 'mock'
ex. mockCandidates, mockCriteria